src/bramble/client/main.js

```diff
@@ -750,7 +750,14 @@ define([
                         // If the path was a file, immediately unlink
                         // trigger the event, and call the callback
                         if(err.code === "ENOTDIR") {
-                            return shell.rm(path, genericFileEventFn("fileDelete", path, callback));
+                            return shell.rm(path, genericFileEventFn("fileDelete", path, function(err) {
+                                wrappedCallback = callback;
```

Comment:
this is missing a `var` - without it, this variable will be globally scoped, not function scoped, and very unpredictable things will happen if this triggers twice.